### DISCUSSION TITLE
Whitelisting another Plugin

### DISCUSSION THREAD

**Discussion by Drachsi** [Cookie-AutoDelete/Cookie-AutoDelete#1040] [Q&A / Support] [answered]
The Plugin interferes with the loading of a website, The Plugin supplier requests I whitelist this ( https://wordpress.org/support/topic/cookies-and-the-plugin-version-5-82/ ) This is for FireFox and windows 10 The site is at [](https://www.northampton-business-directory.com/)

How do I do that please. I am not technical so would appreciate help.

Regards
Drachsi

**Comment by kennethtran93** [comment, marked as answer]
You need to whitelist the site that you have visited to which the plugin is running on. All Cookies created are normally linked to the site visited.

Visit the website that uses the plugin, then click on the browser icon for this web extension to open a popup, then click 'whitelist' next to the site. The one without the '*.' will only keep cookies from that site (e.g. wordpress.org only.  not from blog.wordpress.org), while the one with '*.' will keep cookies including its subdomains. (e.g. wordpress.org and including blog.wordpress.org)

**Reply by Drachsi** [reply]
Hi, Thanks for your help. I am lost.
First part ok
I then clicked on tools Addons then on Cookie Autodelete  Clicked settings then manage. Cannot find the popup with "whitelist" listed anywhere.

Regards
Drachsi

**Reply by kennethtran93** [reply]
Apologies for the delay in responding. 

The popup isn't in the settings. If you're going through that route, then once you're in the setting, you can navigate to 'List of Expressions' tab. From there you will need to enter in the url used for that site, then click whitelist. 

An easier way is the popup, which is accessible through our icon on the toolbar.  Make sure you are on the tab which has that site open. If you click on our icon, it opens the pop up, with the url of that site filled in. From there you can simply click whitelist.

**Reply by Drachsi** [reply]
Hi, Thanks for your reply. I did it through the icon route, that was nice and easy.
Many thanks, stay safe.
Regards
Drachsi
